include/mbedtls/config.h

```diff
@@ -502,16 +507,19 @@
  * function, but will use your mbedtls_internal_ecp_double_jac if the group is
  * supported (your mbedtls_internal_ecp_grp_capable function returns 1 when
  * receives it as an argument). If the group is not supported then the original
- * implementation is used. The other functions and the definition of
- * mbedtls_ecp_group and mbedtls_ecp_point will not change, so your
- * implementation of mbedtls_internal_ecp_double_jac and
- * mbedtls_internal_ecp_grp_capable must be compatible with this definition.
+ * implementation is used, unless disabled by MBEDTLS_ECP_NO_FALLBACK. The
+ * other functions and the definition of mbedtls_ecp_group and
+ * mbedtls_ecp_point will not change, so your implementation of
+ * mbedtls_internal_ecp_double_jac and mbedtls_internal_ecp_grp_capable must be
+ * compatible with this definition.
  *
  * Uncomment a macro to enable alternate implementation of the corresponding
  * function.
  */
 /* Required for all the functions in this section */
 //#define MBEDTLS_ECP_INTERNAL_ALT
+/* Turn off software fallback for curves not supported in hardware */
+//#define MBEDTLS_ECP_NO_FALLBACK
```

Comment:
For mbetls 3.0, we plan to change negative options to positive ones (see #4034 ) thus better to do it that way here. Thus,
MBEDTLS_ECP_HAVE_FALLBACK instead of MBEDTLS_ECP_NO_FALLBACK.

Reply:
Done.

Reply:
But we have to use a negative option now for backward compatibility. Users can write their own standalone `config.h` that doesn't refer to the distributed one. This is pretty common in embedded applications. Upgrading Mbed TLS should not break code that has its own standalone `config.h`.

(After 3.0, we should be able to add new positive options thanks to a [new level of indirection](https://github.com/ARMmbed/mbedtls/issues/3960).)

Reply:
Yes sorry, I missed that aspect of the change.

Reply:
So I should revert back to the original version?

Reply:
Indeed, sorry for the noise.

Reply:
Reverted to the original commit. Not sure how to retrigger the testing, since the reported build failures (from the original commit) are due to the test system malfunctioning, and github is smart enough to re-fetch the logs from the original commit instead of triggering a new run.

Reply:
I just retriggered Travis - CI, but you'll have to retrigger the private ARM jobs on your end.

Reply:
I think I know why Travis is failing - I mistakenly pushed the same branch to to this repo instead of my fork, and removed it quickly. I guess there's still some state left in the build system and/or PR that thinks it should look for the PR in the base repo instead of my fork.

Any ideas on how to resolve? Should I just close this PR and open a new one?

Reply:
Don't worry about "Travis CI - Branch". We'll just ignore it.

Reply:
OK, great. Then I think there's nothing else to do on this PR?